internal/infrastructure/mongo/mongodoc/asset_test.go

```diff
@@ -0,0 +1,157 @@
+package mongodoc
+
+import (
+	"github.com/reearth/reearth-backend/pkg/asset"
+	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/bson"
+	"testing"
+	"time"
+)
+
+func TestNewAsset(t *testing.T) {
+	type args struct {
+		asset *asset.Asset
+	}
+
+	newAsset, _ := asset.New().
+		NewID().
+		Team(id.NewTeamID()).
+		Name("test").
+		Size(10).
+		URL("test_url").
+		Build()
+
+	tests := []struct {
+		name  string
+		args  args
+		want  *AssetDocument
+		want1 string
+	}{
+		{
+			name: "new asset",
+			args: args{
+				asset: newAsset,
+			},
+			want: &AssetDocument{
+				ID:          newAsset.ID().String(),
+				CreatedAt:   newAsset.CreatedAt(),
+				Team:        newAsset.Team().String(),
+				Name:        newAsset.Name(),
+				Size:        newAsset.Size(),
+				URL:         newAsset.URL(),
+				ContentType: newAsset.ContentType(),
+			},
+			want1: newAsset.ID().String(),
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			got, got1 := NewAsset(tt.args.asset)
+			assert.Equalf(t, tt.want, got, "NewAsset(%v)", tt.args.asset)
+			assert.Equalf(t, tt.want1, got1, "NewAsset(%v)", tt.args.asset)
```

Comment:
```suggestion
			assert.Equal(t, tt.want, got)
			assert.Equal(t, tt.want1, got1)
```

If this asserts fail, it is not necessary to output it yourself, since the differences between objects are clearly displayed.